PTerm-UI/TerminalEmulatorMorph.class.st

```diff
@@ -1251,6 +1251,7 @@ TerminalEmulatorMorph >> processKeyEvent: evt [
 	"Handle VT100 control key"
 	(evt controlKeyPressed)
 			ifTrue: [
+				(char = 32) ifTrue: [ ^ down downcall: 0 ].
```

Comment:
So we have the same problem on Pharo 7 (master). We need to synchronise the common code between the two branches.

I will integrate this modification on master, then rebase this branch to master.